samples/CoreAnimationTest/CoreAnimationTest/ProgrammaticViewController.m

```diff
@@ -38,13 +38,13 @@ - (void)viewDidLoad {
     [self addMenuItemViewController:[[CALayerAnimationViewController alloc] init] andTitle:@"CALayer Animation (CAAnimation)"];
 
     // CAShapeLayer
-    [self addMenuItemSegue:@"CAShapeLayer" andTitle:@"CAShapeLayer"];
+    [self addMenuItemSegue:@"CAShapeLayer" andTitle:@"CAShapeLayer - not supported"];
 
     // CAScrollLayer
-    [self addMenuItemSegue:@"CAScrollLayer" andTitle:@"CAScrollLayer"];
+    [self addMenuItemSegue:@"CAScrollLayer" andTitle:@"CAScrollLayer - not supported"];
```

Comment:
Terry and I discussed this - there is value in showing how we compare. I can go either way.

---
In reply to: [99437929](https://github.com/Microsoft/WinObjC/pull/1874#discussion_r99437929) [](ancestors = 99437929)